packages/datagateway-dataview/src/page/pageContainer.component.tsx

```diff
@@ -477,51 +479,57 @@ class PageContainer extends React.Component<
   };
 
   public render(): React.ReactElement {
-    return (
-      <Paper square elevation={0} style={{ backgroundColor: 'inherit' }}>
-        <NavBar
-          entityCount={this.props.entityCount}
-          cartItems={this.props.cartItems}
-          navigateToSearch={this.props.navigateToSearch}
-          navigateToDownload={this.props.navigateToDownload}
-        />
-
-        <StyledGrid container>
-          {/* Toggle between the table and card view */}
-          <Grid item xs={12}>
-            <Route
-              exact
-              path={this.state.paths}
-              render={() => (
-                <CardSwitch
-                  toggleCard={this.state.toggleCard}
-                  handleToggleChange={this.handleToggleChange}
-                />
-              )}
-            />
-          </Grid>
+    // Renders the homepage if the current location equals /datagateway
+    // TODO determine if this is the best way of doing things
+    if (this.props.location.pathname === paths.homepage) {
+      return <HomePage />;
```

Comment:
May be a better way of rendering the homepage. This is very plugin-specific too. It was done this way because the entirety of the dataview package seems to revolve around the PageContainer component. Right now, it is essentially saying if we are on /datagateway, load the homepage, otherwise load the normal PageContainer page as default.